modules/osdk-ansible-k8s-module-testing-locally.adoc

```diff
@@ -40,7 +40,7 @@ This example creates and deletes a namespace with the switch of a variable.
 [source,yaml]
 ----
 - name: set test namespace to {{ state }}
```

Comment:
Get the below errors by follwing this doc:
```
[root@preserve-olm-env foo-operator]# ansible-playbook playbook.yml 
[WARNING]: No inventory was parsed, only implicit localhost is available

[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

ERROR! couldn't resolve module/action 'community.kubernetes.k8s'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/data/goproject/src/github.com/example-inc/foo-operator/roles/foo/tasks/main.yml': line 3, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

# tasks file for foo
- name: set test namespace to "{{ state }}"
  ^ here
We could be wrong, but this one looks like it might be an issue with
missing quotes. Always quote template expression brackets when they
start a value. For instance:

    with_items:
      - {{ foo }}

Should be written as:

    with_items:
      - "{{ foo }}"
```

```
[root@preserve-olm-env foo-operator]# operator-sdk version
operator-sdk version: "v0.17.1", commit: "6d108056f39fa19546cf9235fad6e84b0683114a", kubernetes version: "unknown", go version: "go1.13.10 linux/amd64"
[root@preserve-olm-env foo-operator]# ansible --version
ansible 2.9.0
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.6.8 (default, Sep 26 2019, 11:57:09) [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
```